package-scripts.js

```diff
@@ -55,7 +66,19 @@ module.exports = {
       'start-server': 'electron dist/electron --server',
       'start-electron': 'NODE_ENV=development electron dist/electron',
       'builder-dist': 'electron-builder --mac -p never',
-      'builder-dist-linux': 'electron-builder --linux -p never'
+    },
+    linux: {
```

Comment:
Linux and mac support are more or less the same. Do you think it makes sense to create "unix" section and have linux and mac scripts delegate to "unix"?

Reply:
I'll tackle this in a followup PR